variables.tf

```diff
@@ -0,0 +1,35 @@
+variable "aws_region" {
```

Comment:
I should have added descriptions and type. will add them

Reply:
Try adding validation rules 😜